GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Objects/GameData.cs

```diff
@@ -44,8 +43,10 @@ public class GameData
     private const int InitialCoinAmount = 20;
     private const int TotalCarCount = 4;
     private const int TotalBackgroundCount = 2;
+    private PlayerController _playerController =  Object.FindObjectOfType<PlayerController>();
+    private GameObject _backGroundImages = GameObject.Find("background/backGroundImages").gameObject;
```

Comment:
Nit: backGround as a capitalization is a bit odd, the g should probably be lowercase either in background or Background

Reply:
oh that's a typo, fixed it!